### ISSUE TITLE
Option label being saved instead of value

### ISSUE THREAD

**Issue by bshamblen** [aldeed/meteor-autoform-select2#28]
I have a fairly basic implementation of a select2 field, where I'm setting the `options` attribute using a helper function, to dynamically generate the list of select options. Until the most recent update this was working as expected, exactly the same as a regular select field but with autocomplete functionality. The `label`s would be displayed to the user and the selected `value` would be saved in the mongoDB record. 

Now, instead of the `value` being saved the `label` is being saved. I've added the references to the the select2 .js and .css files in my page template, per the instructions for the 2.0.0 upgrade, but it's still saving the `label`, not the `value`. Here's an example of how I have it configured now, but reduced to just a static options list (mine is pulling from a collection). 

``` html
{{>afQuickField name='fieldName' type='select2' options=selectOptions}}
```

My helper function looks something like this:

``` javascript
Template.myTemplateName.helpers({
    selectOptions: function() {
        return [
            {
                label: 'Item 1',
                value: '1'
            },
            {
                label: 'Item 2',
                value: '2'
            },
            {
                label: 'Item 3',
                value: '3'
            },
            {
                label: 'Item 4',
                value: '4'
            },
            {
                label: 'Item 5',
                value: '5'
            }
        ];
    }
})
```

If I select "Item 4" from the list and submit the form the value of `fieldName` is set to "Item 4", instead of "4", like a standard select field. Again, this worked correctly until the most recent update. Any help would be greatly appreciated. Thanks!

**Comment by benjamin79** [comment]
I´ve the same error. Ended up using a normal html select and adding data-schema-key="yourkey". Now it works.

**Comment by aldeed** [comment]
Please make sure you're using the latest select2 (4.0.0-rc.2) and the latest of autoform and of this pkg (2.0.2). If you still have this problem, provide a link to a reproduction repo. I couldn't reproduce.

**Comment by tom2strobl** [comment]
I have the same issue, updated to 2.0.2 but it still doesn't show the values. I use this in my schema:

    type: {
        type: String,
        optional: true,
        autoform: {
          type: "select2",
          afFieldInput: {
            firstOption: "Select a Type"
          },
          options: function(){
            return [
              {label: "Available as a Freelancer", value: "freelancer"},
              {label: "Employee at a Partner Company", value: "partner"},
              {label: "Employee at a Client Company", value: "client"},
              {label: "Applicant for WILD Jobs", value: "applicant"},
              {label: "Miscellaneous", value: "misc"}
            ];
          }
        }
    },

and it doesn't render the values in the select field

    <select firstoption="Select a Type" name="type" data-schema-key="type" class="form-control select2-offscreen" tabindex="-1" title="">
        <option>Available as a Freelancer</option>
        <option>Employee at a Partner Company</option>
        <option>Employee at a Client Company</option>
        <option>Applicant for WILD Jobs</option>
        <option>Miscellaneous</option>
    </select>

So when one saves it, it saves the option text as there is no specified value.

**Comment by tom2strobl** [comment]
I had the same problem and I can confirm that updating everything fixes the issue. I first had the nauser-select2 package in there which has an old version of select2 and then I had several autoform plugins requiring an older version of autoform. Bringing Select2 on 4.0.0 and autoform on 5.0.0 fixed the issue for me.

**Comment by bshamblen** [comment]
I created a sample project with just one form, one field and it worked as expected. Going back to my other project I performed a `meteor update` thinking that would solve everything, but apparently one of the other form types that we're using `yogiben:autoform-tags` still requires auto form 4.0. So, I'm stuck for the moment, but not as a result of this issue. Thanks for your help.

**Comment by aldeed** [comment]
@bshamblen, you can clone `yogiben:autoform-tags` repo into your /packages folder and change the `use` call in `package.js` to `'aldeed:autoform@4.2.2 || 5.0.0'`. As a temporary workaround until @yogiben has time to publish a compatible release.

**Comment by bshamblen** [comment]
@aldeed Thanks, I'll give that a try.

**Comment by bshamblen** [comment]
This has been resolved with an upgrade to 5.0. Thanks!
